load_collection.json

```diff
@@ -268,13 +303,19 @@
                                     }
                                 }
                             }
-                        }
+                        },
+                        "crs": 3857
                     },
                     "result": true
                 }
             }
         }
     ],
+    "exceptions": {
+        "TargetCrsMissing": {
+            "message": "The data requested from the collection has multiple CRS assigned and thus a valid target CRS must be specified."
+        }
```

Comment:
Even when the data has multiple CRSs assigned, the backend can probably do a better job of selecting the more appropriate option than the user. Forcing the backend to fail when CRS is not provided is IMO counter-productive.